packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 48.15.0 - May 14, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-lwc
+
+- Added a better error message when starting the Local Dev server and another process is running on the same port ([PR #2172](https://github.com/forcedotcom/salesforcedx-vscode/pull/2172))
```

Comment:
Nit pick - Can we make it "Updates to error message" instead of "Added a better".